vignettes/troubleshooting.Rmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "Troubleshooting"
+author: "KimYen Ladia"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Troubleshooting}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+## Troubleshooting
+
+For users who are using a Windows machine and a network mapped drive, where your folder mapped from a drive `H:\\Documents\\R\\win-library\\3.4` to a network location `\\\\samba-xxx/home/<your_home_dir>/Documents/R/win-library/3.4`, you may experience the following error:
```

Comment:
where your folder mapped from -> where the folder is mapped from a drive, e.g., `H:\\Documents\\R\\win-library\\3.4` to a network location, e.g., `\\\\samba-xxx/home/<your_home_dir>/Documents/R/win-library/3.4`